cmd/create/cluster/cmd.go

```diff
@@ -1217,7 +1218,7 @@ func run(cmd *cobra.Command, _ []string) {
 
 	var availabilityZones []string
 	if useExistingVPC || subnetsProvided {
-		subnets, err := awsClient.GetSubnetIDs()
+		subnets, err := awsClient.GetSubnetIDs(&ec2.DescribeSubnetsInput{})
```

Comment:
1. Where do you pass non-empty input?
2. I don't think it's nice to create aws specific object outside of the aws client and pass it to an aws client func

Reply:
If https://github.com/openshift/rosa/pull/765/files#diff-39260fbf46ea076702be0085fc20b3897395b779af4dbaee85902ce0b95d725dR363 is the only place you need the filter, which is internal to aws client I suggest the following.

Make `getSubnetIDs(&ec2.DescribeSubnetsInput{})` private and keep `GetSubnetIDs()` without the parameter
Then, from both `GetSubnetIDs()` and from `getVPCSubnets` call `getSubnetIDs`

Reply:
Makes sense, changed, thanks a lot for this solution.